index.html

```diff
@@ -3,184 +3,196 @@
+
+<!-- Triple image set -->
+<div class="width_capped">
+    <div class="container">
+        <div id="triple_image_wrapper">
+        <article class="card" role="article">
+            <a class="triple_a" href="/about/">
+            <div class="card-text">
+                <h2 class="card-title">About us</h2>
             </div>
-            <div class="col-12 col-lg-6">
-                <p>Since 1976, <strong>Computer Science House</strong> has provided a revolutionary living and learning environment for its members. With unique facilities, an emphasis on hands-on learning, and a strong social atmosphere, CSH helps its members grow as professionals and more.</p>
-                <a class="btn btn-primary" href="/about/">More about CSH</a>
+            <img class="card-image" src="assets/images/disk.png" />
+            </a>
+        </article>
+        </div>
+
+        <div id="triple_image_wrapper">
+        <article class="card" role="article">
+            <a class="triple_a" href="/about/projects.html">
+            <div class="card-text">
+                <h2 class="card-title">Our projects</h2>
             </div>
+            <img class="card-image" src="assets/images/drink.png" />
+            </a>
+        </article>
         </div>
-    </div>
+
+        <div id="triple_image_wrapper">
+        <article class="card" role="article">
+            <a class="triple_a" href="/about/tour.html">
+            <div class="card-text">
+                <h2 class="card-title">Visit CSH</h2>
+            </div>
+            <img class="card-image" src="assets/images/project_room.png" />
+            </a>
+        </article>
+        </div>
+    </div>              
 </div>
 
-<div class="container spaced" id="post-list">
-    <div class="row">
-        <div class="col-12">
-            <h2 class="header">Recent posts</h2>
+<div  style="background-color: #eaeafa;">
+    <div class="width_capped">
+        <div class="dual_container">
+            <p class="dual_container_item" id="intro_blurb">Since 1976, Computer Science House has 
+                provided a revolutionary living and learning 
+                environment for its members. With unique 
+                facilities, an emphasis on hands-on learning, 
+                and a strong social atmosphere, CSH helps its 
+                members grow as professionals and more.</p>
+                <!-- TODO: Make the thing below a carousel with older group pictures -->
+            <img class="dual_container_item" src="assets/images/welcome_back_large.webp" alt="CSH stands together on a grass field"/>
+        </div>                
+    </div>                
+</div>
+
+<!-- Projects tab -->
+<h3 class="left_offset" style="padding-top: 0.6em;">Things we're working on ...</h3>
+<div class="quad_image">
+    <div class="content">
+        <a href="csh.rit.edu">
```

Comment:
this 404s

Reply:
Removed links from this section cause there's nowhere to link to